build.sbt

```diff
@@ -24,7 +22,8 @@ Test / scroogeThriftSourceFolder := { baseDirectory {
 Test / scroogeThriftOutputFolder := (Test / sourceManaged).value
 Test / managedSourceDirectories += (Test / scroogeThriftOutputFolder).value
 
-scalacOptions++= Seq("-deprecation", "-unchecked", "-release:11")
```

Comment:
Apparently `-unchecked` does this:

> -unchecked or --unchecked
> Enable additional warnings where generated code depends on assumptions.

We removed it while we were desperately trying to figure out what the solution to the Java version issue was, and probably don't _need_ to remove it, but I'm not sure what benefit we were getting from it!